scripts/validation.js

```diff
@@ -0,0 +1,58 @@
+function isValidEmail(email) {
+  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+  return emailRegex.test(email);
+}
+
+function isValidPasswordLength(password, minLength = 8) {
+  return password.length >= minLength;
+}
```

Comment:
💊 제안
추후 확작성을 고려하셔서 작성해주신 것은 좋으나 이렇게 되면 요구사항상 비밀번호 min값인 8에 대한 명확성이 코드상 떨어집니다.
저라면 minLength는 필요할때 추가하시거나 8이라는 수를 상수로 가지고 계셔서 조금 더 명확하게 작성하시는 것을 추천드려요.

```suggestion
export const MIN_PASSWORD_LENGTH = 8;
function isValidPasswordLength(password, minLength) { // default 값으로는 사용처에서 알 수 없음.
  return password.length >= minLength;
}

const isPasswordValid = isValidPasswordLength(input, MIN_PASSWORD_LENGTH);
```